daprdocs/content/en/operations/security/mtls.md

```diff
@@ -199,6 +199,17 @@ kubectl rollout restart statefulsets/dapr-placement-server -n <DAPR_NAMESPACE>
 
 4. Restart your Dapr applications to pick up the latest trust bundle.
 
+{{% alert title="Recommended CLI command to rotate certificates with new ones" color="success" %}}
```

Comment:
It is hard to separate out the kubectl approach and the Dapr CLI approach here. Can you add headings for each so that these are separated? For example "Root certificate upgrade using kubectl" and "Root certificate upgrade using CLI"

Reply:
Also my concern here, is that given the CLI is the preferred method we should have this first BEFORE the kubectl method. So could you switch this around in the order?

Reply:
I am now keeping these in separate section. Kindly have a look at recent changes.